pkg/microservice/aslan/core/workflow/service/workflow/pipeline_validation.go

```diff
@@ -523,9 +525,13 @@ func releaseCandidate(b *task.Build, taskID int64, productName, envName, imageNa
 		customTarRule = project.CustomTarRule
 	}
 
+	if len(commitID) > 8 {
```

Comment:
It is suggested to use a constant  to reflect the semantics of `maximum length` by constant name.